src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
It's a good practice to set box-sizing to border-box globally for all elements at the root level of your styles, to ensure consistent layout calculations across all elements. Consider moving this to a global stylesheet to index.scss

Reply:
please move this `index.scss` file and use it for every element:

```
* {
  box-sizing: border-box;
}
```